aiohasupervisor/root.py

```diff
@@ -109,12 +109,19 @@ async def info(self) -> RootInfo:
         result = await self._client.get("info")
         return RootInfo.from_dict(result.data)
 
+    async def reload_updates(self) -> None:
+        """Reload updates.
+
+        Reload main components update information (OS, Supervisor, Core and plug-ins).
+        """
+        await self._client.post("reload_updates", timeout=ClientTimeout(total=300))
+
     async def refresh_updates(self) -> None:
-        """Refresh updates."""
+        """Refresh updates (discouraged)."""
```

Comment:
I'd instead add a body to the docstring, ie a blank line and then a new paragraph after the first sentence, which explains that this call is discouraged and why that is.